--- ion-devel-20020413/system.mk~	Mon Apr 29 18:58:25 2002
+++ ion-devel-20020413/system.mk	Mon Apr 29 18:59:51 2002
@@ -85,8 +85,8 @@
 	-Wtrigraphs -Wformat -Wchar-subscripts \
 	-Wparentheses -pedantic-errors -Wuninitialized
 
-
-CFLAGS=-g -O2 $(WARN) $(DEFINES) $(INCLUDES) $(EXTRA_INCLUDES)
+OPTFLAGS=-g -O2
+CFLAGS=$(OPTFLAGS) $(WARN) $(DEFINES) $(INCLUDES) $(EXTRA_INCLUDES)
 LDFLAGS=-g $(LIBS) $(EXTRA_LIBS)
 
 
