--- ion-devel-20030225/src/main.c.orig	Sat Feb 22 21:36:35 2003
+++ ion-devel-20030225/src/main.c	Tue Feb 25 23:18:53 2003
@@ -116,7 +116,7 @@
 	wglobal.argc=argc;
 	wglobal.argv=argv;
 	
-	if(!wmcore_init("ion-devel", ETCDIR, LIBDIR, display, onescreen))
+	if(!wmcore_init("ion", ETCDIR, LIBDIR, display, onescreen))
 		return EXIT_FAILURE;
 	
 	init_funclists();
--- ion-devel-20030225/Makefile.orig	Tue Feb 25 23:22:54 2003
+++ ion-devel-20030225/Makefile	Tue Feb 25 23:22:59 2003
@@ -23,7 +23,7 @@
 DOCS= 	README LICENSE ChangeLog \
 	doc/config.txt doc/functions.txt doc/draw.txt doc/query.txt
 
-IONETCDIR=$(ETCDIR)/ion-devel
+IONETCDIR=$(ETCDIR)/ion
 
 ######################################
 
