--- ion-devel-20020413/system.mk~	Mon Apr 29 18:51:05 2002
+++ ion-devel-20020413/system.mk	Mon Apr 29 18:54:30 2002
@@ -7,11 +7,11 @@
 ## Installation paths
 ##
 
-PREFIX=/usr/local/ion-devel
+PREFIX=$(DESTDIR)/usr/X11R6
 
 # No need to modify these usually
 BINDIR=$(PREFIX)/bin
-ETCDIR=$(PREFIX)/etc
+ETCDIR=$(DESTDIR)/etc/X11
 MANDIR=$(PREFIX)/man
-DOCDIR=$(PREFIX)/doc
+DOCDIR=$(PREFIX)/share/doc
 # Not used
 INCDIR=$(PREFIX)/include
 LIBDIR=$(PREFIX)/lib
