--- ion-devel-20030225/system.mk.orig	Tue Feb 25 23:28:28 2003
+++ ion-devel-20030225/system.mk	Tue Feb 25 23:29:19 2003
@@ -7,13 +7,13 @@
 ## Installation paths
 ##
 
-PREFIX=/usr/local/ion-devel
+PREFIX=$(DESTDIR)/usr
 
 # No need to modify these usually
 BINDIR=$(PREFIX)/bin
-ETCDIR=$(PREFIX)/etc
+ETCDIR=$(DESTDIR)/etc/X11
-MANDIR=$(PREFIX)/man
+MANDIR=$(PREFIX)/share/man
-DOCDIR=$(PREFIX)/doc
+DOCDIR=$(PREFIX)/share/doc
 LIBDIR=$(PREFIX)/lib
 INCDIR=$(PREFIX)/include
 
